app/assets/stylesheets/vendor/main.css

```diff
@@ -3434,7 +3434,7 @@ div.timeline-year {
 }
 
 .map article.moveOut, .slidePopOut.moveOut, .timeContainer.moveOut {
-    margin-right: -106%;
+    margin-right: -106% !important;
```

Comment:
This CSS applied to the mobile view when the facet is showing.  While I don't generally like to use `!important`, it was the simplest way I could find to override line 4205 (see notes below for explanations about the use of margins).